index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+thebroadwalk.com
+        </title>
+        <style>
```

Comment:
move the files to external CSS file --- [using external css](https://www.w3schools.com/css/css_howto.asp)